2007-05-16  Geoffrey Keating  <geoffk@apple.com>

	* unwind-dw2.c (uw_identify_context): Use the CFA, not the IP.

Index: gcc/unwind-dw2.c
===================================================================
--- gcc/unwind-dw2.c	(revision 124749)
+++ gcc/unwind-dw2.c	(working copy)
@@ -1526,7 +1526,7 @@
 static inline _Unwind_Ptr
 uw_identify_context (struct _Unwind_Context *context)
 {
-  return _Unwind_GetIP (context);
+  return _Unwind_GetCFA (context);
 }
 
 
